snap/gadget.go

```diff
@@ -27,73 +27,43 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-type gadgetYaml struct {
-	Volumes map[string]volume `yaml:"volumes,omitempty"`
+type GadgetInfo struct {
+	Volumes map[string]GadgetVolume `yaml:"volumes,omitempty"`
+
+	// Default configuration for snaps (snap-id => key => value).
```

Comment:
Yeah, gadgets always make me feel like something that won't be updated very often, so snap-id is the safest. Having them in the output of `snap info` should be enough.